blog/_posts/2021-11-29-tilt-news-november-2021.md

```diff
@@ -0,0 +1,74 @@
+But for now, the November edition comes packed with great reads, awesome features, and one or two glimpses behind the scenes!
+
+## Tilt Mentions
+We love to see more and more folks mentioning and discussing Tilt publicly! 
+
+* Clear Street’s Ottavio Hartman wrote a blog post on [leveraging Tilt for their local dev environment](https://medium.com/@ottopo/523045dd2ac8) dubbed “localkube”.
+* In “[Defining the Developer Experience](https://dzone.com/articles/developer-experience)”, Kevin Taylor, VP of Engineering at Caylent, ponders the implications of Cloud Native development on the evolution of DevX. Tilt is mentioned together with Skaffold as tools to help develop on Kubernetes.
+* In his book [Effortless Cloud-Native App Development Using Skaffold](https://books.google.com/books?id=vNJCEAAAQBAJ&pg=PA233&lpg=PA233&dq=effortless+cloud-native+app+development+using+skaffold+tilt&source=bl&ots=NUwNtLM0Ci&sig=ACfU3U38wdnpWJF78uxTiNKkbgYDSnHAhQ&hl=en&sa=X&ved=2ahUKEwjx1Z3jnYn0AhXPSt8KHVsCBnMQ6AF6BAgTEAM#v=onepage&q=effortless%20cloud-native%20app%20development%20using%20skaffold%20tilt&f=false), Ashish Choudhary introduces Tilt as a possible alternative to Skaffold, 
+
+Hopefully one day we’ll be able to add *your* take on Tilt too. Whether your experience with Tilt was superb or awful, we love to read about it! 🤓
+
+## Tilt News
+### Docs
+Long-time Tilters have probably already noticed that we have overhauled our [docs](http://docs.tilt.dev) and introduced a [new, shiny tutorial](https://docs.tilt.dev/tutorial/index.html).  
+So far, our docs were mainly a collection of information by maintainers. From now on, we are designing our docs with our users in mind first. Rather than giving you all the information you could possibly need (and more), we’re trying to understand which problems you are trying to solve and how to best guide you through the solution finding process.  
+To read more about the changes, [visit our blog](https://blog.tilt.dev/2021/11/08/new-tutorial.html).
+
+### Custom Builds
+Often people ask us for help to optimize their docker builds. Unfortunately, some issues lie in the nature of the beast. However, did you know there are plenty of alternatives out there that work with Tilt?  
+Check out [Nick’s blog post](https://blog.tilt.dev/2021/11/12/docker-build-alternatives.html) to learn how to use `kubectl build`, `ko`, and `pack` with Tilt!
+
+### How-To Button 2: The Rebuttoning
+There are sequels nobody wants, but everyone knows are coming, like the tenth installment of “The Fast and the Furious”. Then there are sequels no one expected, but once they’re out you can’t imagine the world without them, like the exceptional [Terminator 2: Judgement Day](https://www.youtube.com/watch?v=nxr2SV5znwI).
```

Comment:
```suggestion
There are sequels nobody wants, but everyone knows are coming, like the tenth installment of “The Fast and the Furious.” Then there are sequels no one expected, but once they’re out you can’t imagine the world without them, like the exceptional [Terminator 2: Judgement Day](https://www.youtube.com/watch?v=nxr2SV5znwI).
```

Reply:
American English is weird

Reply:
if you're ever in the mood for pain this is *chef's kiss* https://www.chicagomanualofstyle.org/home.html